app/src/main/java/com/mk/tasky/agenda/data/local/AgendaDao.kt

```diff
@@ -0,0 +1,23 @@
+package com.mk.tasky.agenda.data.local
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.mk.tasky.agenda.data.local.entity.ReminderEntity
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface AgendaDao {
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun insertReminder(reminder: ReminderEntity)
+
+    @Query(
+        """
+            SELECT * FROM ReminderEntity
+            WHERE dateTime > :dayOne
```

Comment:
Very nitpick and will never cause any harm, but just in terms of logical correctness it should be >= here, right? Because 00:00:00 would be part of the current day 😄

Reply:
Interesting
Fixed!